README.md

```diff
@@ -24,6 +24,7 @@ A curated list of companies using Elixir in production, organized by industry.
 #### Gaming
 
 * [Undead Labs](http://undeadlabs.com/) ([GitHub](https://github.com/undeadlabs)) - Creators of the smash hit survival-fantasy game _State of Decay_ and forthcoming tactical creature battler _Moonrise_.
+* [22cans](http://www.22cans.com/) ([GitHub](https://github.com/22cans)) - 22cans is an independent game studio founded by the industry legend and creator of God Game genre, Peter Molyneux. Creators of Curiosity and Godus.
```

Comment:
Hi @mr-bt, thanks for the addition!  Would you mind moving `22cans` to before `Undead Labs`?  I was hoping to keep the list in alphabetical order (with numbers and symbols first).